vagrant/provision.sh

```diff
@@ -87,4 +87,5 @@ echo Dashboard token:
 kubectl -n kube-system describe secret default| awk '$1=="token:"{print $2}'
 echo
 
-ln -s /sumologic/vagrant/sumologic.sh /usr/local/bin/sumologic
+ln -s /sumologic/vagrant/scripts/sumomake.sh /usr/local/bin/sumomake
```

Comment:
Can we call it `sumo-make` instead of `sumomake`?

I bet we'll be having more of those command starting with `sumo-` and IMO it will be much cleaner.